targets/simple_switch/simple_switch.cpp

```diff
@@ -501,17 +501,14 @@ SimpleSwitch::ingress_thread() {
     Field &f_egress_spec = phv->get_field("standard_metadata.egress_spec");
     port_t egress_spec = f_egress_spec.get_uint();
 
-    Field &f_clone_spec = phv->get_field("standard_metadata.clone_spec");
-    unsigned int clone_spec = f_clone_spec.get_uint();
+    unsigned int clone_mirror_session_id =
```

Comment:
do those need to be `unsigned int` or can we just use `auto` and use whichever type is returned by the register access proxy

Reply:
Will try replacing all of those types with `auto` and see how it goes.